### DISCUSSION TITLE
Is possible to check if some method is used?

### DISCUSSION THREAD

**Discussion by jaychang0917** [LemonAppDev/konsist#787] [Ask a Question]
### Wanted end result.

May i know how to check if some method is used? For example, is possible to know if `Bar.b()` is used in the `Foo` class?

```
class Foo {
	fun some() {
		val bar = Bar()
		bar.b("hello")
	}
}

class Bar {
	fun b()
}
```

### What you tried so far.

Document seems doesn't have API to do that

**Comment by igorwojda** [comment]
This is impossible to do ATM. We are considering utilizing K2 Compiler to handle such cases, but it will take some time to even verify if this is possible.
